aggregator_test.go

```diff
@@ -94,3 +95,30 @@ func TestStartStorageConnection(t *testing.T) {
 		t.Fatal("Cannot create storage object", err)
 	}
 }
+
+func TestLoadConfigurationOverrideFromEnv(t *testing.T) {
+	err := os.Unsetenv("INSIGHTS_RESULTS_AGGREGATOR_CONFIG_FILE")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	main.LoadConfiguration("tests/config1")
+
```

Comment:
I'd add Os.Unsetenv() just to be sure that other tests won't affect this one

Reply:
actually, yes, it's better to clean the env completely before the test. Thanks